Library/Homebrew/download_strategy.rbi

```diff
@@ -1,13 +1,14 @@
 # typed: strict
 
-# This is a third-party implementation
+# This is a third-party implementation.
 # rubocop:disable Lint/StructNewOverride
```

Comment:
Might be worth putting in a `gems/*.rbi` location where we can disable this globally with RuboCop,

Reply:
"Resolving" this comment so this can merge, but I'll do this in a separate PR.